test/Interop/Cxx/function/template-irgen-objc.swift

```diff
@@ -0,0 +1,52 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+
+// TODO: This needs to be fixed in the SwiftTypeConverter.
+// Currently "Any" is imported as an Objective-C "id".
+// That doesn't work unless we have Objective-C interop.
+// Once that's done, this test can be merged with "template-irgen".
+// REQUIRES: objc_interop
+
+import FunctionTemplates
+
+// CHECK-LABEL: define {{.*}}void @"$s4main18testPassThroughAny1xypyp_tF"(%Any* noalias nocapture sret %0, %Any* noalias nocapture dereferenceable(32) %1)
+
+// CHECK: [[ANY_ALLOC:%.*]] = alloca %Any
```

Comment:
Do we really need all these checks in this test to prove the point? I'd be ok with only having the line 39 (the one with `_Z11passThroughIP11objc_objectET_S2_`)

Reply:
You're right, we don't need all that.